### PR TITLE
Add mosaic visualization

### LINKED ISSUE #1290: Document the mosaic visualization type
PR: https://github.com/influxdata/influxdb/pull/19292

There is a new "mosaic" visualization type that looks like this:

image

##### Relevant URLs
- https://v2.docs.influxdata.com/v2.0/visualize-data/visualization-types/

### PR DESCRIPTION
**Pull request by noramullen1**
- Closes #1290 
- Also updates wording and un-comments out legend on band doc (unrelated to new feature)

### CODE REVIEW

File: content/influxdb/v2.0/visualize-data/visualization-types/mosaic.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+---
+title: Mosaic visualization
+list_title: Mosaic
+list_image: /img/influxdb/2-0-visualizations-mosaic-example.png
+description: >
+  The Mosaic view displays the single value most recent value for a time series in a mosaic view.
```
Comment (russorat) [inline content/influxdb/v2.0/visualize-data/visualization-types/mosaic.md:6]:
```suggestion
  The Mosaic visualization displays data from two or more qualitative variables.
```

File: content/influxdb/v2.0/visualize-data/visualization-types/mosaic.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+---
+
+The **Mosaic** visualization displays data from two or more qualitative variables.
+
+{{< img-hd src="/img/influxdb/2-0-visualizations-mosaic-example.png" alt="mosaic example" />}}
+
+Select the **Mosaic** option from the visualization dropdown in the upper left.
+
+## Mosaic behavior
+The mosaic visualization displays string values in a graph.
```
Comment (russorat) [inline content/influxdb/v2.0/visualize-data/visualization-types/mosaic.md:21]: Not sure i understand this line or section. The mosaic allows you to view state changes in your time series data visually.

File: content/influxdb/v2.0/visualize-data/visualization-types/mosaic.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+
+## Mosaic controls
+To view **Mosaic** controls, click **{{< icon "gear" >}} Customize** next to the visualization dropdown.
+
+###### Data
+- **Fill Column**: Select a column to fill in the mosaic tiles.
+- **X Column**: Select a column to display on the x-axis.
+- **Y Column**: Select one or more columns to display on the y-axis.
+- **Time Format**: Select the time format. Options include:
```
Comment (russorat) [inline content/influxdb/v2.0/visualize-data/visualization-types/mosaic.md:31]: @TCL735 is this now updated with many more? maybe we just link to a page with all the formats on them?

Comment (TCL735) [reply content/influxdb/v2.0/visualize-data/visualization-types/mosaic.md:31]: Yes. Time Format now has many more options. We can address this in a separate PR that updates this on all pages for all graphs types.

File: content/influxdb/v2.0/visualize-data/visualization-types/mosaic.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+weight: 206
+menu:
+  influxdb_2_0:
+    name: Mosaic
+    parent: Visualization types
+---
+
+The **Mosaic** visualization displays data from two or more qualitative variables.
+
+{{< img-hd src="/img/influxdb/2-0-visualizations-mosaic-example.png" alt="mosaic example" />}}
```
Comment (sanderson) [inline content/influxdb/v2.0/visualize-data/visualization-types/mosaic.md:16]:
```suggestion
{{< img-hd src="/img/influxdb/2-0-visualizations-mosaic-example.png" alt="Mosaic data visualization" />}}
```

(For the SEO)

File: content/influxdb/v2.0/visualize-data/visualization-types/mosaic.md (hunk 3 above)
Comment (sanderson) [reply content/influxdb/v2.0/visualize-data/visualization-types/mosaic.md:31]: @noramullen1 We could create a shortcode for the list of time values so we only have to edit it in one place. Doesn't have to happen in this PR, but it'd be good to make this a little more maintainable.

File: content/influxdb/v2.0/visualize-data/visualization-types/mosaic.md [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+    - **Total Tick Marks**: Enter the total number of timestamp ticks to display.
+    - **Start Tick Marks At**: Enter the time, in RFC3339 format, to start displaying ticks. Use the **Date Picker** field to automatically generate an RFC3339 formatted timestamp for this field.
+    - **Tick Mark Interval**: Enter the number of milliseconds in between each timestamp tick.
+
+###### Y Axis
+- **Y Axis Label**: Enter a label for the y-axis.
+- **Y Label Separator**: If there's more than one column on the y-axis, enter a delimiter to separate the label, such as a comma or space. If there's no separator specified, the labels are a continuous string of all y columns.
+- **Generate Y-Axis Tick Marks**: Select the method to generate y-axis tick marks:
+  - **Auto**: Select to automatically generate tick marks.
+  - **Custom**: To customize the number of y-axis tick marks, select this option, and then enter  the following:
```
Comment (sanderson) [inline content/influxdb/v2.0/visualize-data/visualization-types/mosaic.md:61]:
```suggestion
  - **Custom**: To customize the number of y-axis tick marks, select this option, and then enter the following:
```

**Review by sanderson — APPROVED** [review]
Just some minor suggestions

File: content/influxdb/v2.0/visualize-data/visualization-types/mosaic.md (hunk 3 above)
Comment (noramullen1) [reply content/influxdb/v2.0/visualize-data/visualization-types/mosaic.md:31]: Sounds good, created an issue: https://github.com/influxdata/docs-v2/issues/2249

**Comment by TCL735** [comment]
@noramullen1 "Y Label Columns" is missing. In Giraffe, I describe it as:

"column names that are used for labeling each tick in the y-axis and the header inside the legend. This allows the y-axis labels and legend header to be less verbose than naming all of the columns that used for y. Does not affect data grouping. This property is a subset of y (see above). Any elements in yLabelColumns that do not appear in y are ignored."

Let me know if that makes sense to you.

**Pull request merged**